client/src/utils/nettkurs-utils.ts

```diff
@@ -18,9 +18,17 @@ export enum Tags {
 }
 
 export const IAVideoer: QbrickVideo[] = [
```

Comment:
For nå er dette greit, men hvis det kommer mange flere videoer så må vi finne en måte å hente listen automatisk fra Qbrick.